Core/Utilities/Time.py

```diff
@@ -264,10 +264,10 @@ def queryTime(f):
   """ Decorator to measure the function call time
   """
   def measureQueryTime(*args, **kwargs):
-    start = time.time()
+    start = nativetime.time()
     result = f(*args, **kwargs)
     if result['OK'] and not 'QueryTime' in result:
-      result['QueryTime'] = time.time() - start
+      result['QueryTime'] = nativetime.time() - start
```

Comment:
Fixed also in https://github.com/DIRACGrid/DIRAC/pull/2712 (commit https://github.com/fstagni/DIRAC/commit/aa51b019631592f95d7b2f4024965e9d564f025b)